pos_accounting/src/pool/pool_data.rs

```diff
@@ -70,6 +70,10 @@ impl PoolData {
         self.pledge_amount = Amount::ZERO;
         self
     }
+
+    pub fn is_decommissioned(&self) -> bool {
+        self.pledge_amount == Amount::ZERO
```

Comment:
after rebasing on master this has to check `staker_balance` which is pledge + rewards